### PR TITLE
Par unseq/unseq adding

### LINKED ISSUE #5492: Create missing task execution policies
Notably, `unseq` and `par_unseq` don't expose corresponding `task` policies.

### PR DESCRIPTION
**Pull request by hcq9102**
**1. Add policies:**

    unsequenced_task_policy;
    unsequenced_task_policy_shim;
    unsequenced_policy;
    unsequenced_policy_shim;
    parallel_unsequenced_task_policy;
    parallel_unsequenced_task_policy_shim;
    parallel_unsequenced_policy;
    parallel_unsequenced_policy_shim;

      Add tests for unseq/par_unseq in rotate.cpp 

**2. Add implementations for new policies to handle local exceptions**

      Add tests in rotate.cpp to test if that works

Fixes #5492

### CODE REVIEW

**Comment by hcq9102** [comment]
Have no idea why some commits come over from my own hpx repository. 
For creating this PR today(5.16), I only add the two latest commits from my local workstation. 
(Looks like those unrelated commits are not interfering with existing files anyway)

**Comment by hkaiser** [comment]
> Have no idea why some commits (7 days ago, 21days ago) come over from my own hpx repository. 

You might want to rebase onto master, that should get rid of all superfluous commits.

File: libs/core/executors/include/hpx/executors/execution_policy.hpp [hunk 1]
```diff
@@ -1548,6 +1548,24 @@ namespace hpx { namespace execution {
             return params_;
         }
 
+        /// \cond NOINTERNAL
```
Comment (hkaiser) [inline libs/core/executors/include/hpx/executors/execution_policy.hpp:1551]: If you add a `/// \cond ...` you should add a `/// \endcond` as well.

**Comment by hkaiser** [comment]
@msimberg the clang-apex-builder on Daint reports a strange error claiming that there is a version control header in one of the files, even if it isn't in the file in the repository. I'm stumped - would you have any ideas?

**Comment by hkaiser** [comment]
The HIP builder failures are caused by configuration issues on Rostam. We will fix those separately

**Review by hkaiser — APPROVED** [review]
LGTM, thanks!

**Comment by hkaiser** [comment]
bors merge

**Pull request merged**